docs/roles/secretary.rst

```diff
@@ -9,22 +9,33 @@ This page explains more about the responsibilities and role of the secretary.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The secretary is responsible for scheduling, advertising, and recording RITlug general and eboard meetings.
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from campus? working remote? is under academic suspension (i.e. low GPA)? >
+* Must be a full time student on main campus for at least one semester
+* Cannot be on academic probation
+* Must be present for **75% of weekly club meetings and eboard meetings**
 
 
 Responsibilities
 ----------------
 
-< responsibilities and routine work that someone in this position is directly accountable for – what do they do and what should other eboard members expect from this person? >
+* *Weekly Presentations*
+   * Review of presentations
+   * Addition of presentations to RITlug website
+* Record meeting minutes during both general and eboard meetings
```

Comment:
Do we take meeting minutes at general meetings, or do we want to start doing this? If it isn't something we are doing today, let's omit the general meetings.

Reply:
I think you're right in that notes for general meetings aren't going to provide value.

For eboard meetings, I think it's a necessity, and something we do today, so I'll modify this.